Mobile emulation for Chrome browser

nirmalashish opened the pull request.

anandbagmar commented:
@nirmalashish - a few suggestions:
1. Can we add a new test for mobile web to show how this will be used?
2. Update the README

anandbagmar commented:
@nirmalashish - yes, I understand the logic is in the downstream. Hence a working test to show how it can be used will be helpful.

nirmalashish commented:
@anandbagmar 
- added a new feature file that tells user how to pass mobile emulation specific parameters; also added guidance at step def level
- updated README for this new feature

anandbagmar commented on README.md:
```diff
@@ -139,6 +139,18 @@ You can run these tests as below:
+
+Example:
+
+![mobile-emulation-on-web-browser.png](mobile-emulation-on-web-browser.png)
```
this png seems to be missed for checking

nirmalashish replied:
added now (previously ignored due to gitignore)

anandbagmar commented on src/main/resources/default_browser_config.json:
```diff
@@ -51,11 +55,19 @@
       "headless": true
-    }
+    },
+    "tab1" : "iPad",
```
though we have added the values for firefox and safari, this is not yet implemented in Drivers, right?

nirmalashish replied:
yes, the logic is only present for Chrome as of now
